apps/web/src/components/Calendar/body/day/calendar-body-day-calendar.tsx

```diff
@@ -13,12 +13,12 @@ export default function CalendarBodyDayCalendar () {
   const today = new Date()
   const { date, events, setDate } = useCalendarContext()
 
-  const [hideGoToButton, setHideGoToButton] = useState(sameDay(date, today))
```

Comment:
dont we need to remove this include of sameDay above?

Reply:
Its not removed, its function has just been relocated and renamed